sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Release History
 
-## 5.0.0-beta.1 (Unreleased)
+## 5.0.0-beta.1 (2021-02-09)
```

Comment:
Nit (not blocking at all): are you releasing tonight yet, or tomorrow? Still, day really won't matter much.

Reply:
I'm releasing now - I was already through the pipeline when it failed because of the missing date.